starknet_py/net/account/account_client.py

```diff
@@ -469,6 +471,28 @@ async def execute(
     async def deploy(self, transaction: Deploy) -> DeployTransactionResponse:
         return await self.client.deploy(transaction=transaction)
 
+    def create_deployer(
```

Comment:
I think we can remove this method to avoid problems with typing, pyright can't handle string types :|